server/auth_server.go

```diff
@@ -967,6 +968,23 @@ func GetRancherAPIHost() string {
 	return apiHost
 }
 
+//GetRedirectWhitelist reads the auth.service.redirect.whitelist setting
+func GetRedirectWhitelist() string {
+	var settings []string
+
+	//add the setting
+	settings = append(settings, authServiceRedirectWhitelistSetting)
+	dbSettings, err := readCommonSettings(settings)
+	if err != nil {
+		log.Errorf("GetRedirectWhitelist: Error reading DB setting %v", err)
+		return ""
+	}
+	whitelist := dbSettings[authServiceRedirectWhitelistSetting]
+	log.Debugf("GetRedirectWhitelist() returning %v ", whitelist)
+
+	return whitelist
+}
```

Comment:
Since we are always reading this setting from the DB ad not caching it, we need not add this new property to the list of properties to watch and reload auth-service on in cattle.

Reply:
i.e. a config reload is not needed if this setting change.

Reply:
Makes sense to remove from reload list, but should we also read this from cache?

Reply:
yes I think its an overhead to read this from DB on every user login and we should cache it

Reply:
yep, good catch :)